src/components/features/Order/OrderMessage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { useFormContext } from 'react-hook-form';
+import { Textarea } from '@chakra-ui/react';
+import GiftDetail from '../GiftDetail';
+
+const MAX_MESSAGE_LENGTH = 100;
+
+export default function OrderMessage() {
+  const { register } = useFormContext();
+
+  return (
+    <OrderMessageContainer>
+      <Message>
+        <Title>나에게 주는 선물</Title>
+        <Textarea
+          placeholder="선물과 함께 보낼 메시지를 적어보세요"
+          {...register('message')}
+          maxLength={MAX_MESSAGE_LENGTH}
```

Comment:
```suggestion
          {...register('message', { maxLength: MAX_MESSAGE_LENGTH })}
```

해당 방향도 가능하긴 한데, 가급적이면 useForm의 register option으로 전달해주는게 확실한 방법이라 개선해보는게 좋겠습니다.